packages/nitro-protocol/contracts/test/TestErc20AssetHolder.sol

```diff
@@ -0,0 +1,13 @@
+pragma solidity ^0.5.13;
+pragma experimental ABIEncoderV2;
+import '../ERC20AssetHolder.sol';
+
+/**
+  * @dev This contract is a clone of the ERC20AssetHolder contract. It will be initialized to point to the TestNitroAdjudicator.
+*/
+contract TestErc20AssetHolder is ERC20AssetHolder {
+    constructor(address _AdjudicatorAddress, address _TokenAddress)
+        public
+        ERC20AssetHolder(_AdjudicatorAddress, _TokenAddress)
+    {}
+}
```

Comment:
See #602 which also fixes the issue that led to this file being written.

Reply:
As far as I can see, the separate ganache instances also solve this, so these changes might not be necessary here at all.